dash-docs/protocol-documentation.md

```diff
@@ -318,7 +318,7 @@ Get Masternode list or specific entry
 
 | Field Size | Field Name | Data type | Description |
 | ---------- | ----------- | --------- | ---------- |
-| 41 | vin | [CTxIn](#ctxin) | The unspent output which is holding 1000 DASH
+| 36 | outpoint | [COutPoint](#coutpoint) | The unspent output which is holding 1000 DASH
```

Comment:
Nit: Field name is `masternodeOutpoint` in most other messages

Reply:
Yep, this one is used only temporary in actual code and can be changes to `masternodeOutpoint` with no consequences to readability, I agree.